support/camel-k-maven-plugin/src/main/java/org/apache/camel/k/tooling/maven/GenerateCatalogMojo.java

```diff
@@ -483,18 +488,32 @@ private void addCapabilities(RuntimeSpec.Builder runtimeSpec, CamelCatalogSpec.B
         artifacts.add(Artifact.from("org.apache.camel.quarkus", "camel-quarkus-opentracing"));
         addCapabilityAndDependecies(runtimeSpec, catalogSpec, "tracing", artifacts, false);
 
+        // Telemetry capability
         artifacts.clear();
         artifacts.add(Artifact.from("org.apache.camel.quarkus", "camel-quarkus-opentelemetry"));
-        addCapabilityAndDependecies(runtimeSpec, catalogSpec, "telemetry", artifacts, false);
-
-        artifacts.clear();
-        artifacts.add(Artifact.from("org.apache.camel.k", "camel-k-master"));
-        addCapabilityAndDependecies(runtimeSpec, catalogSpec, "master", artifacts, true);
+        properties.clear();
+        properties.add(Property.from("endpoint", "quarkus.opentelemetry.tracer.exporter.otlp.endpoint"));
+        properties.add(Property.from("serviceName", "quarkus.opentelemetry.tracer.resource-attributes"));
+        properties.add(Property.from("sampler", "quarkus.opentelemetry.tracer.sampler"));
+        properties.add(Property.from("samplerRatio", "quarkus.opentelemetry.tracer.sampler.ratio"));
+        properties.add(Property.from("samplerParentBased", "quarkus.opentelemetry.tracer.sampler.parent-based"));
```

Comment:
it would be nice to validate if i.e. the same keys also apply to spring-boot

Reply:
Correct. I think this would be the next step when we onboard the new runtime. We'd need to understand how to harmonize the parameters expected by the operator according to the different runtimes requirements. The goal of this PR is to move towards that goal without breaking compatibility of what's expected now. Eventually it will be a matter of changing these parameters in the future catalog releases, driven by the changes in the operator as well.

Reply:
I guess there are many more questions that must be answered, i.e. :
- how to handle cases where a key is not applicable, but another one is needed  
- how to handle breaking changes, so properties that have disappeared or become required ('d like to avoid having to do version checks)